R/pkg/DESCRIPTION

```diff
@@ -42,6 +42,7 @@ Collate:
     'functions.R'
     'install.R'
     'jvm.R'
+    'mllib_wrapper.R'
```

Comment:
No. Even if it wasn't automatically generated by `roxygen`, we have to enforce loading base case classes first.